governator-core/src/main/java/com/netflix/governator/guice/InternalLifecycleModule.java

```diff
@@ -40,52 +36,42 @@
 import com.netflix.governator.lifecycle.LifecycleListener;
 import com.netflix.governator.lifecycle.LifecycleManager;
 import com.netflix.governator.lifecycle.LifecycleMethods;
+import com.netflix.governator.lifecycle.LifecycleMethodsFactory;
 import com.netflix.governator.lifecycle.warmup.DAGManager;
 
 class InternalLifecycleModule extends AbstractModule {
-    // this really serves as a Set purpose.
-    // put dummy boolean as Map value.
-    // value is really not important here.
-    private final CopyOnWriteArraySet<Dependency<?>> seen = new CopyOnWriteArraySet<Dependency<?>>();
-
-    private final LoadingCache<Class<?>, LifecycleMethods> lifecycleMethods = CacheBuilder
-        .newBuilder()
-        .softValues()
-        .build(new CacheLoader<Class<?>, LifecycleMethods>(){
-            @Override
-            public LifecycleMethods load(Class<?> key) throws Exception {
-                return new LifecycleMethods(key);
-            }});
-    
-    private final AtomicReference<LifecycleManager> lifecycleManager;
-
-    InternalLifecycleModule(AtomicReference<LifecycleManager> lifecycleManager) {
-        this.lifecycleManager = lifecycleManager;
-    }
 
-    @Override
-    public void configure() {
-        bindListener(
-            Matchers.any(),
-            new TypeListener(){
-                @Override
-                public <T> void hear(final TypeLiteral<T> type, TypeEncounter<T> encounter) {
-                    encounter.register(
-                        new InjectionListener<T>() {
-                            @Override
-                            public void afterInjection(T obj) {
-                                processInjectedObject(obj, type);
-                            }
+    public static class LifecycleInjectionListener implements TypeListener {
+
+        // this really serves as a Set purpose.
+        // put dummy boolean as Map value.
+        // value is really not important here.
```

Comment:
Is this comment just out of date? I'm guessing a map was being used as set in some previous version.